packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -363,3 +388,22 @@ function validateManifestId(manifest: ManifestInfo) {
     throw new Error(`The manifest file doesn't contain the id of the Office Add-in.`);
   }
 }
+
+export async function webView(manifestPath: string, webViewString?: string) {
```

Comment:
While the command name is `webView`, we could choose a more descriptive name for the exported function. Also, we shouldn't have the console.log() statements be in the exported function. I think what would be better to is to have separate functions setWebViewType() and getWebViewType() and take the code which checks if webViewString is truthy or not and does set or get and move that to the command implementation.